src/Uno.Extensions.Authentication.MSAL/Uno.Extensions.Authentication.MSAL.WinUI.csproj

```diff
@@ -63,8 +61,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Localization.Abstractions" />
-		<PackageReference Include="Microsoft.Identity.Client" Condition="'$(IsMSALSupported)'=='true'" />
-		<PackageReference Include="Microsoft.Identity.Client.Extensions.Msal" Condition="'$(IsMSALSupported)'=='true'" />
+		<PackageReference Include="Microsoft.Identity.Client" />
```

Comment:
We should double check that these dependencies don't cause compilation errors (in Release) with any of the Uno targets